### PR TITLE
[CLEANUP] Rector: Add return type void to function like without any return

### PR DESCRIPTION
**Pull request by ziegenberg**
This applies the rule **AddVoidReturnTypeWhereNoReturnRector**. For Details see:
https://github.com/rectorphp/rector/blob/main/docs/rector_rules_overview.md#addvoidreturntypewherenoreturnrector

### CODE REVIEW

File: src/CSSList/CSSList.php [hunk 1]
```diff
@@ -263,7 +263,7 @@ public function getLineNo()
      *
      * @return void
```
Comment (oliverklee) [inline src/CSSList/CSSList.php:264]: We can also drop all those `@return void` PHPDoc annotations now.

**Review by oliverklee — CHANGES_REQUESTED** [review]

File: src/CSSList/CSSList.php (hunk 1 above)
Comment (ziegenberg) [reply src/CSSList/CSSList.php]: PHPDoc annotations are gone.

**Review by JakeQZ — COMMENTED** [review]
Should the Rector config file be updated - specifically to uncomment `AddVoidReturnTypeWhereNoReturnRector::class`?

**Review by oliverklee — APPROVED** [review]

**Comment by ziegenberg** [comment]
> Should the Rector config file be updated - specifically to uncomment `AddVoidReturnTypeWhereNoReturnRector::class`?

IMHO, this can be done at a later stage. First, I'm doing all the **CLEANUP** and, later, the **TASK** of configuring the rector. Also, @oliverklee surely wants to have TASK and CLEANUP in different PRs.

After this type series, we can set the following config:
```PHP
    ->withSets(
        [
            SetList::TYPE_DECLARATION,
        ]
    )
```

**Pull request merged**